wordpress-mobile/WordPress-iOS
# Remove segue `showSigninV2`

Fixes # n/a

This PR removes the segue `.showSigninV2`, which sends the user to the `SignupEmailViewController`. Now the user is navigated programmatically.

Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/232

### To test:
1. Check out this branch
2. `rake dependencies`
3. Build and run
4. Log out if you're logged in
5. "Sign up for WordPress.com" > "Sign up with Email" > ensure it navigates to the `SignupEmailViewController`. It looks like this:

<img src="https://user-images.githubusercontent.com/1062444/78597567-5ee50300-7813-11ea-96c2-da324646f0c8.png" width="350" />

### Do not merge until:
2. The Authenticator PR has been merged: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/232
3. A new Authenticator release has been created 
4. The release is published on Cocoapods trunk 
5. This PR points to the new release  

PR submission checklist:
